lib/generators/react_on_rails/base_generator.rb

```diff
@@ -67,9 +64,8 @@ def copy_webpack_config
 
       def copy_webpacker_config
         puts "Adding Webpacker v6 config"
-        base_path = "base/base/"
-        base_files = %w[config/webpacker.yml]
-        base_files.each { |file| copy_file("#{base_path}#{file}", file) }
+        base_files = using_shakapacker_7? ? %w[config/shakapacker.yml] : %w[config/webpacker.yml]
+        base_files.each { |file| copy_file("#{base_path}/#{file}", file) }
```

Comment:
why the slash got added?

Reply:
To follow the same structure in creating the file path.

Reply:
Changed back